webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -716,6 +717,12 @@ var userOverride = {
                 if (navigator.onLine) {
                     window.Applozic.ALSocket.reconnect();
                 }
+                SOCKET_RECONNECT_FAIL_COUNT++;
```

Comment:
if the user is offline, then also this message will be displayed along with other offline message? or only 1 will be displayed?

Reply:
It will only display one message as shown in the image 
![Screenshot 2021-12-27 at 9 32 25 AM](https://user-images.githubusercontent.com/19753380/147433144-32999009-b1cc-4752-9c4a-1ac28a977528.png)